src/entity/Source.ts

```diff
@@ -17,6 +17,11 @@ export const COMMUNITY_PICKS_SOURCE = 'community';
 export const SQUAD_IMAGE_PLACEHOLDER =
   'https://daily-now-res.cloudinary.com/image/upload/v1672041320/squads/squad_placeholder.jpg';
 
+export enum SourceType {
+  Machine = 'machine',
+  Squad = 'squad',
+}
```

Comment:
You can actually also set this as the type for `type` it just means we need to adjust it everywhere, but it's worthwhile doing so.

Reply:
I have thought of it but I was afraid it might change the data type to enum (as postgresql supports it), so I kept the original values and types. WDYT?

Reply:
Hmm I've done it for Features now, @idoshamun Do you know if this negatively impacts?
I personally prefer enums, but if it's bad we should not do it.

Reply:
You can use the typescript enum for the variable type and make sure typeorm keeps treating it as text. In this case it has no impact

Reply:
Alrighty. Will carefully do the changes with this one.